spinalcordtoolbox/vertebrae/detect_c2c3.py

```diff
@@ -0,0 +1,116 @@
+# coding: utf-8
```

Comment:
i suggest to add here a documentation that explains how to train a model (for replicability/testing purpose)

Reply:
Note: To re-discuss on Monday.